Added Resources and description to Morgan npm (#2108)
kamranahmedse/developer-roadmap: content/roadmaps/107-nodejs/content/112-nodejs-logging/100-morgan.md

> # Morgan
> 
> Morgan is a NodeJS and expressJS middleware to log the HTTP request and error and it simplifies the debugging process.
> 
> What can Morgan do?
> 
>     * It provides flexibility in defining the format of log messages.
>     * It allows you to overwrite the output destination for your logs.
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink badgeText='Read' colorScheme="yellow" href='https://www.npmjs.com/package/morgan'>Official Website</BadgeLink>
> <BadgeLink badgeText='Read' colorScheme="blue" href='https://www.digitalocean.com/community/tutorials/nodejs-getting-started-morgan'>How to use Morgan|DigitalOcean</BadgeLink>

Reviewer: ```suggestion
# Morgan

Morgan is a NodeJS and express.js middleware to log the HTTP request and error, simplifying the debugging process. It provides flexibility in defining the format of log messages and overwrites the output destination for your logs.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink badgeText='Read' colorScheme="yellow" href='https://www.npmjs.com/package/morgan'>Official Website</BadgeLink>
<BadgeLink badgeText='Read' colorScheme="blue" href='https://www.digitalocean.com/community/tutorials/nodejs-getting-started-morgan'>How to use Morgan|DigitalOcean</BadgeLink>
```
